src/psr4/Bridge.php

```diff
@@ -754,13 +754,21 @@ public function remove_filter( $hook, $mvc_handler, $priority = 10 )
      * Returns class method call mapped to a mvc engine method.
      * @since 1.0.3
      *
+     * @param string|array $call
+     *
      * @return string
      */
     private function get_mapped_mvc_call( $call, $return = false )
     {
-        return ( preg_match( '/[vV]iew\@/', $call ) ? '_v_' : '_c_' )
-            . ( $return ? 'return_' : 'void_' )
-            . $call;
+        if ( is_array( $call ) ) {
+            $class_prefix = $this->config->get( 'namespace' ) . '\\Controllers\\';
```

Comment:
How do you that `/Controllers/` will always be the route?, take into consideration that the path can be changed at `config/app.php`, so this PR is not covering all case scenarios.

Reply:
You are right. I've rewritten this controversial solution.